testing-daml-models/daml-scripts/1_first_script.md

```diff
@@ -0,0 +1,58 @@
+The first thing you need to do when writing a DAML script is to include the DAML script package in
+your dependencies. 
+
+- First, **open the IDE and wait for it to load**
+- After that, click on `daml.yaml`{{open}} to open the `daml.yaml` file.
+- Add the line
+
+  <pre class="file" data-target="clipboard">
+    - daml-script
+  </pre>
+
+  under the `dependencies` stanza. 
+
+Now open the `daml/User.daml`{{open}} file and just before the module declaration line add the
+language pragma
+
+<pre class="file" data-target="clipboard">
+{-# LANGUAGE ApplicativeDo #-}
+</pre>
+
+and just after the module declaration add the `Daml.Script` library to the imports. Your module
```

Comment:
Maybe add the `import Daml.Script` snippet as "Copy to Clipboard"

Reply:
done.